models/nc-news.models.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
+
+const fetchTopics = () => {
+    return db.query("SELECT * FROM topics;").then((result) => {
+        // if (result.rows.length === 0) {
```

Comment:
This result will never be empty, we haven't required any parameters from the user currently and so this will also just give us the response from the SELECT * FROM topics; which is the array of 3 topics. This error you are testing for is not related to this endpoint and so will not pass through the logic of this controller or model. We will need to think about how we can capture all unknown endpoints in our app.js and respond to say that these endpoints don't exist.